practice-01/js/index.js

```diff
@@ -56,8 +80,24 @@ function checkEmailError(input) {
     }
     return isEmailError;
 }
+
+/**
+ * Display an error message when the user enters the wrong email
+ * @param isEmailError : True || False
+ * @param input : Element of email ID
+ */
+function validationEmail(isEmailError, input) {
+    if (isEmailError == false) {
+        showError(input, 'Email Invalid');
+    } else {
+        showSuccess(input);
+    }
+}
+
 /**
- * Validation for charcaters
+ * Check special characters

```

Comment:
Could you explain what is regex used for? 
BTW, `checkCharacter` name is too common since your regex is not a common case